.github/workflows/directory_workflow.yml

```diff
@@ -1,46 +0,0 @@
-name: Directory/Filename Formatter workflow
-on: [push, pull_request]
-
-jobs:
-  main:
-    name: (Directory) Formatter
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@main
-      - name: Setup Git configuration
-        run: |
-          git config --global user.name 'autoprettier'
-          git config --global user.email 'actions@github.com'
-          git remote set-url origin https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/$GITHUB_REPOSITORY
-      - name: Filename Formatter
-        run: |
-          IFS=$'\n'
-          for fname in `find . -type f -name '*.jl' -o -name '*.jl'`
-          do
-            echo "${fname}"
-            if [ "${fname}" == "./src/TheAlgorithms.jl" ]
-            then
-              continue
-            else
-              new_fname=`echo ${fname} | tr ' ' '_'`
-              echo "      ${new_fname}"
-              new_fname=`echo ${new_fname} | tr 'A-Z' 'a-z'`
-              echo "      ${new_fname}"
-              new_fname=`echo ${new_fname} | tr '-' '_'`
-              echo "      ${new_fname}"
-              if [ ${fname} != ${new_fname} ]
-              then
-                echo "      ${fname} --> ${new_fname}"
-                git "mv" "${fname}" ${new_fname}
-              fi
-            fi
-          done
-          git commit -am "Formatting filenames ${GITHUB_SHA::8}" || true
-      - name: Update DIRECTORY.md
-        run: |
-          wget https://raw.githubusercontent.com/TheAlgorithms/scripts/main/build_directory_md.py
-          python3 build_directory_md.py Julia ./src .jl TheAlgorithms.jl > DIRECTORY.md
-
-          git diff
-          git commit -m "Update DIRECTORY.md" DIRECTORY.md || true
-          git push --force origin HEAD:$GITHUB_REF || true
```

Comment:
This seems to be causing the issue. If we push normally, there shouldn't be any issues.

Reply:
Yes. That causes the issue but I don't think there is any point of removing those lines if it does not update the whole repository. That's why I suggest that we do it manually instead and point contributors to use the script from https://github.com/TheAlgorithms/scripts

Reply:
I'll be creating a PR to have the filename formatter and directory workflow without causing any issues. You can see how it works in the C++ repository.

https://github.com/TheAlgorithms/C-Plus-Plus/blob/18f6cef2ad01c03cfffeefa0d4fa709e2145a79c/.github/workflows/awesome_workflow.yml#L25-L34

https://github.com/TheAlgorithms/C-Plus-Plus/blob/18f6cef2ad01c03cfffeefa0d4fa709e2145a79c/.github/workflows/awesome_workflow.yml#L89-L91